src/worker.ts

```diff
@@ -14,8 +25,22 @@ export default {
 			case EVERY_11_HOURS: {
 				return await addMissedUpdatesRole(env);
 			}
-			default:
+
+			case EVERY_1_HOUR: {
+				await syncUsersStatus(env);
+				await syncExternalAccounts(env);
+				await syncUnverifiedUsers(env);
+				await syncIdleUsers(env);
+				await syncNickNames(env);
+				await syncIdle7dUsers(env);
+				await syncOnboarding31dPlusUsers(env);
+				break;
```

Comment:
Will all this run within the time limit?

Reply:
![image](https://github.com/Real-Dev-Squad/cron-jobs/assets/45538392/419b2e0f-72a5-4810-a781-c1d1247d2883)

I am not aware of any time limit for a worker

Reply:
I was thinking for safety reasons we can divide it as two workers

Reply:
<img width="898" alt="Screenshot 2024-03-12 at 8 26 20 PM" src="https://github.com/Real-Dev-Squad/cron-jobs/assets/56365512/ed77ab7b-8ea3-4a8f-b588-025aa3c84847">


Please verify it once from here
https://blog.cloudflare.com/workers-pricing-scale-to-zero/

Reply:
Divided it into two wrokers